docs/releases/v2.0.0.md

```diff
@@ -0,0 +1,13 @@
+# Registry Module version 2.0.0
+
+This is a patch release containing an update to Harbor package.
+
+## Changelog
+
+- Update Harbor from version `v2.2.2` to `v2.4.2`.
+
+## Upgrade path
+
+There's no supported upgrade path for this module from `v1.2.0` to `v2.0.0`, there are several breaking changes in Harbor from v2.2.2 to v2.4.2, especially in [Harbor v2.3.0 the Database schema has changed](https://github.com/goharbor/harbor/releases/tag/v2.3.0), please read carefully the release notes from upstream and plan accordingly.
+
+The recommended approach is a lift and shift migration.
```

Comment:
What is the procedure for the lift and shift?

Reply:
pinging @beratio here. Do we have something written that we can reuse?